python/clients.py

```diff
@@ -0,0 +1,78 @@
+from helpers import read_master_key, ClientBuilder, create_json_schema
+
+"""
+This file is meant to be run to demonstrate CSFLE in action. Prior to running
+this file, ensure you've run make-local-data-key.py
+"""
+
+
+def main():
+
+    local_master_key = read_master_key()
+
+    kms_providers = {
+        "local": {
+            "key": local_master_key,
+        },
+    }
+
+    client_prototype = ClientBuilder(kms_providers=kms_providers)
+
+    # make sure you paste in your base64 key that was generated from
+    # make_data_key.py
+    data_key = None  # replace with your base64 data key!
+
+    schema = create_json_schema(data_key=data_key)
+    csfle_enabled_client = client_prototype.get_csfle_enabled_client(schema)
+    regular_client = client_prototype.get_regular_client()
+
+    example_document = {
+        "name": "Jon Doe",
+        "ssn": 241014209,
+        "bloodType": "AB+",
+        "medicalRecords": [
+            {
+                "weight": 180,
+                "bloodPressure": "120/80"
+            }
+        ],
+        "insurance": {
+            "provider": "MaestCare",
+            "policyNumber": 123142
+        }
+    }
+
+    regular_client_patients_coll = regular_client.get_database(
+        "medicalRecords").get_collection("patients")
+
+    csfle_client_patients_coll = csfle_enabled_client.get_database(
+        "medicalRecords").get_collection("patients")
+
+    # performing the insert operation with the csfle enabled client
```

Comment:
Nitpick: did you mean "perform the insert"

Reply:
From reading your following code comments, it looks like you meant to use the present participle which implies present continuous tense. I usually comment using either an imperative or simple present tense, e.g.:
imperative - "Perform the insert"
simple present - "Performs the insert"